makefile-sysdig

```diff
@@ -5,16 +5,18 @@ ARTIFACTORY := docker.internal.sysdig.com
 IMAGE_REPO := linux-bench-dependency
 IMAGE_TAG?= dev
 DEPENDENCY_IMAGE_NAME := $(ARTIFACTORY)/$(IMAGE_REPO):$(IMAGE_TAG)
-DEPENDENCY_IMAGE_NAME_linux_arm64 := $(ARTIFACTORY)/$(IMAGE_REPO)_linux_arm64:$(IMAGE_TAG)
+DEPENDENCY_IMAGE_NAME_linux_s390x := $(ARTIFACTORY)/$(IMAGE_REPO)_linux_s390x:$(IMAGE_TAG)
 
 build-dependency-image:
 	$(DOCKER) build -t $(DEPENDENCY_IMAGE_NAME) .
-	$(DOCKER) build -t $(DEPENDENCY_IMAGE_NAME_linux_arm64) -f Dockerfile_linux_arm64 .
+	$(DOCKER) run --rm --privileged multiarch/qemu-user-static --reset -p yes
+	$(DOCKER) buildx create --node multiarch --use
+	$(DOCKER) buildx build --platform linux/s390x --load -t $(DEPENDENCY_IMAGE_NAME_linux_s390x) -f Dockerfile_linux_s390x .
 
 push-dependency-image:
 	$(DOCKER) push $(DEPENDENCY_IMAGE_NAME)
-	$(DOCKER) push $(DEPENDENCY_IMAGE_NAME_linux_arm64)
+	$(DOCKER) push $(DEPENDENCY_IMAGE_NAME_linux_s390x)
 
 delete-dependency-image:
 	$(DOCKER) rmi $(DEPENDENCY_IMAGE_NAME)
-	$(DOCKER) rmi $(DEPENDENCY_IMAGE_NAME_linux_arm64)
\ No newline at end of file
+	$(DOCKER) rmi $(DEPENDENCY_IMAGE_NAME_linux_s390x)
```

Comment:
Should we have both ARM and zLinux here?

Reply:
Yup retested ARM and fixed the command